File: algorithm/src/main/java/baekjoon/p10829/Main.java
package baekjoon.p10829;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println(solve(sc.next()));
    }

    private static String solve(String input) {
        StringBuilder sb = new StringBuilder();
        long n = Long.parseLong(input);

        while (n != 0) {
            sb.append(n % 2);
            n /= 2;
        }

        return sb.reverse().toString();
    }

}

File: algorithm/src/main/java/baekjoon/p10829/Main.kt
package baekjoon.p10829

fun main(args: Array<String>) {
    println(solve(readLine()!!.toLong()))
}

private fun solve(input: Long): String {
    return buildString {
        var n = input
        while (n != 0L) {
            insert(0, n % 2)
            n /= 2
        }
    }
}